lib/msf/core/payload/android.rb

```diff
@@ -65,11 +68,13 @@ def generate_config(opts={})
     config
   end
 
-  def sign_jar(jar)
+  def sign_jar(jar, new_key)
```

Comment:
Hm; Maybe drop the `new_key` argument here until there's a valid use case for it? 🤔 

Just thinking: If we let this becomes configurable, and every android payload starts generating its own key - then we're back to the same position as before.

Reply:
This is true ^^  I'll save it for when it's 100% **needed**